examples/adder/src/lib.rs

```diff
@@ -29,9 +29,16 @@ impl Adder {
         &self,
         #[callback_unwrap] a: DoublePair,
         #[callback_unwrap] b: DoublePair,
-        #[callback_vec] others: Vec<DoublePair>,
```

Comment:
`#[callback_vec]` isn't really compatible with `#[callback_unwrap]` to be used in this manner, due to a feature??/bug??, because it revisits the same promise indices, already visited by `callback_unwrap`, 
so `others` vector in this example contained `a` and `b` as first elements